### ISSUE TITLE
[feature request] JSX support

### ISSUE THREAD

**Issue by weakish** [chejen/eslint-plugin-i18n#5]
It seems text in JSX elements/components is not detectable:

```jsx
<Divider horizontal>高级选项</Divider>
```

Here "高级选项" are Chinese characters, but `no-chinese-character` will not detect them.

**Comment by chejen** [comment]
Hi @weakish, let's make the transpiler do its job. Please consider applying [`babel-eslint`](https://github.com/babel/babel-eslint) to your `.eslintrc`'s [`parser`](https://eslint.org/docs/user-guide/configuring#specifying-parser) option, and it should work fine.

**Comment by weakish** [comment]
Thanks. `babel-eslint` works but it has some issues on some other TypeScript syntax (parsing errors and false positive). `"parser": "@typescript-eslint/parser"` does not work.

**Comment by chejen** [comment]
@weakish Sorry for late response. Do you mind providing some sample code for your use case?

**Comment by weakish** [comment]
The following patterns are considered problems:

```jsx
<Divider horizontal>高级选项</Divider>
```

The following patterns are not considered problems:

```jsx
<Trans i18nKey="pagination.display" values={{ total: formatNumber(count) }}>
  共 <b /> 条
</Trans>
```

**Comment by chejen** [comment]
@weakish Could you make a demo project with your configuration on Github or zip it as an attachment? Both cases that you provide work on my side.

**Comment by chejen** [comment]
Btw, I'm gonna add JSX support this weekend, @weakish you can wait until then.

**Comment by weakish** [comment]
Here is a demo project:  [weakish/react-hello-world](https://github.com/weakish/react-hello-world/blob/master/src/App.tsx) The relevant demo file is `src/App.tsx`.

**Comment by chejen** [comment]
@weakish There is a new release, please give it a try, thanks.

**Comment by weakish** [comment]
@chejen  Thanks a lot! I tested it on a production project (private repository) and it works great!

The following patterns are currently considered problems with eslint-plugin-i18n.
But this is acceptable for implementation simplicity and I really should have use English text in Trans component. 

```ts
return <Trans i18nKey="hello"><button>你好， {{name}}</button></Trans>
```
